Interni/StudioDiFattibilità/components/fattibilità-c2.tex

```diff
@@ -0,0 +1,45 @@
+\section{Capitolato C2}
+
+\subsection{Informazioni generali}
+\begin{itemize}
+\item \textbf{Nome:} \textit{EmporioLambda}:  piattaforma di e-commerce in stile Serverless;
+\item \textbf{Proponente:} Red Babel;
+\item \textbf{Committenti:} \textit{Prof. \Tullio{}} e \textit{Prof. \Riccardo{}}.
+\end{itemize}
+
+\subsection{Descrizione del capitolato}
+Il capitolato propone la realizzazione di un e-commerce, queste piattaforme sono uniche dal punto di vista del percorso che effettua l'utente: comincia dal website/app entrando nella home, passando attraverso i vari prodotti, arrivando alla lista degli oggetti scelti, ed infine alla pagina di pagamento. 
```

Comment:
La descrizione va a  dare la definizione generica che potrebbe andare in un glossario di un e-commerce, senza dire nulla sul progetto realizzato. Da cambiare assolutamente.